src/main/kotlin/no/nav/familie/ef/sak/integration/PdlClient.kt

```diff
@@ -108,6 +109,9 @@ class PdlClient(val pdlConfig: PdlConfig,
                                                                                      dataMapper: (DATA) -> T?): T {
 
         if (pdlResponse.harFeil()) {
+            if (pdlResponse.errors?.any { it.extensions?.notFound() == true } == true) {
```

Comment:
Trenger vi `== true` på slutten her?

Reply:
Ja, `pdlResponse.errors?.any { it.extensions?.notFound() == true }` kan være null pga `errors?`
Hadde kunnet endre det slik att errors er default emptylist

Reply:
Ja, glemte å tenke på spørsmålstegnet der ja...

Reply:
Går det an å legge denne sjekken i pdlResponse?

`pdlResponse.harNotFoundFeil()` - så er det lettere å se hva som skjer her?